src/ds/d400/d400-device.cpp

```diff
@@ -950,6 +951,27 @@ namespace librealsense
             register_info(RS2_CAMERA_INFO_USB_TYPE_DESCRIPTOR, usb_type_str);
 
         std::string curr_version= _fw_version;
+
+        register_features();
```

Comment:
This is very dangerous... I made the same mistake just recently.
We cannot call virtual functions from inside the ctor.

Reply:
Good catch...
Removed the `virtual` property.